gvm/xml.py

```diff
@@ -91,17 +93,45 @@ def pretty_print(xml):
             List[lxml.etree.Element] or directly a lxml element.
 
     """
+    if file is not sys.stdout:
+        if isinstance(file, str):
+            file = Path(file)
+        try:
+            file.is_file()
+        except IsADirectoryError as e:
+            raise e from None
+        if not file.exists():
+            file.touch()
+        file = open(file, 'w')
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.open
```suggestion
        file = file.open('w')
```
or
```python
file = open(str(file), 'w')
```
to support python 3.5.

Reply:
Yeah I have seen that the 3.5 tests failed ... :(